hasura/schema/user/schema.graphql

```diff
@@ -254,32 +255,6 @@ type GenerateApiKeyResponse {
   hash: String!
 }
 
-type GuildAdmin {
-  address: String!
-}
-
-input GuildInfoInput {
```

Comment:
I think we don't want these to be removed...

@CryptoGraffe we want this for users too?

Reply:
@CryptoGraffe It looks like a perms issue is preventing these from coming through on the user level.